examples/remix/README.md

```diff
@@ -0,0 +1,181 @@
+# Use with Remix
+
+This example demonstrates how to use `jest-preview` with [`Remix`](https://remix.run/), bootstrapped using [`create-remix`](https://remix.run/docs/en/v1#getting-started) (without using any built-in templates/stacks).
+
+🚧 This example only demonstrates the simplest use of CSS in a Remix app (global stylesheet). Remix supports [many different ways of styling](https://remix.run/docs/en/v1/guides/styling). Example of more complex ways to style is coming.
+
+## Install Jest
+
+You have to manually install Jest in your Remix app.
+
+First, install `jest` itself. If you are using `jest` version 28 or later, you must install `jest-environment-jsdom` too ([reference](https://github.dev/facebook/jest/blob/585854fca63851a891c5be3e1381c04e341210e1/packages/jest-resolve/src/utils.ts#L116)).
```

Comment:
I think you should use `github.com` instead of `github.dev`

Reply:
Updated